misc_python/jobparser.py

```diff
@@ -112,17 +182,19 @@ def __init__(self, data, scheduled = False):
         else:
             self.Weekday = None
             self.Day = struct.unpack("<H", data[4:6])[0]
-            self.Hour = struct.unpack("<H", data[12:14])[0]
-            self.Minute = struct.unpack("<H", data[14:16])[0]
-            self.Second = struct.unpack("<H", data[16:18])[0]
-            self.Milliseconds = struct.unpack("<H", data[18:20])[0]
+            self.Hour = 00
```

Comment:
Why did you make these zero ?

Reply:
Because Schedule Star Date and Schedule End Date Do not contain hours, minuts, second and milliseconds. It was added to keep the formating the same without having to write seperate class to handle those dates.

Reply:
What you have listed as Scheduled Date is actually two dates consisting of year, month, day, year, month, day.
https://msdn.microsoft.com/en-us/library/cc248290.aspx